personalsite/src/App.js

```diff
@@ -0,0 +1,94 @@
+import './App.css';
+
+function App() {
+  
+  const buttonFunction = () => {
```

Comment:
awesome idea for using the button. This looks great and it's always good to know how to programmatically change css :)

Reply:
but soon you'll see how we can do this without having to access the elements through the document. pretty snazzy stuff coming up soon